sunpy/net/jsoc/tests/test_jsoc.py

```diff
@@ -15,6 +15,9 @@
 from sunpy.net.jsoc import JSOCClient, JSOCResponse
 from sunpy.util.exceptions import SunpyUserWarning
 
+# Ensure all JSOC tests are run on the same parallel worker
+pytest.mark.xdist_group(name="jsoc")
```

Comment:
See [pytest docs](https://docs.pytest.org/en/6.2.x/example/markers.html#marking-whole-classes-or-modules). Running `pytest --co -m "xdist_group"` shows this marks the whole module.
```suggestion
pytestmark = pytest.mark.xdist_group(name="jsoc")
```